File: VB/LocalizeApplication/MauiProgram.vb
Imports DevExpress.Maui

Namespace LocalizeApplication

    Public Module MauiProgram

        Public Function CreateMauiApp() As MauiApp
            Dim builder = MauiApp.CreateBuilder()
            builder.UseMauiApp(Of App)().UseDevExpress(useLocalization:=True).ConfigureFonts(Function(fonts)
                fonts.AddFont("OpenSans-Regular.ttf", "OpenSansRegular")
                fonts.AddFont("OpenSans-Semibold.ttf", "OpenSansSemibold")
            End Function)
            Return builder.Build()
        End Function
    End Module
End Namespace

File: CS/LocalizeApplication/MauiProgram.cs
﻿using DevExpress.Maui;

namespace LocalizeApplication;

public static class MauiProgram {
	public static MauiApp CreateMauiApp() {
		var builder = MauiApp.CreateBuilder();
		builder
		.UseMauiApp<App>()
        	.UseDevExpress(useLocalization: true)
			.UseDevExpressCollectionView()
			.UseDevExpressControls()
         	.ConfigureFonts(fonts =>
		{
			fonts.AddFont("OpenSans-Regular.ttf", "OpenSansRegular");
			fonts.AddFont("OpenSans-Semibold.ttf", "OpenSansSemibold");
		});
        return builder.Build();
	}
}
